packages/blade/src/components/Skeleton/__tests__/Skeleton.native.test.tsx

```diff
@@ -20,4 +21,18 @@ describe('<Skeleton />', () => {
     );
     expect(toJSON).toMatchSnapshot();
   });
+
+  it('should be hidden', () => {
```

Comment:
probably in future PR we can create a util test that we can just include in all the test by default by passing the name of the attribute so we don't have to write this again and again

Reply:
Sure, opened an issue to track https://github.com/razorpay/blade/issues/1443